formSkeleton/forms/Eredienst-notulen/form.ttl

```diff
@@ -0,0 +1,61 @@
+
+
+########### Eredienst notulen ###########
+
+# Conditionally add the form fields to the "type dossier" field
+fields:0827fafe-ad19-49e1-8b2e-105d2c08a54a form:hasConditionalFieldGroup fields:27c26cbe-4f46-4404-b937-6f0d56ca2700.
+
+# Only add the fields if the user selected the "Notulen" option
+fields:27c26cbe-4f46-4404-b937-6f0d56ca2700 a form:ConditionalFieldGroup ;
+    mu:uuid "27c26cbe-4f46-4404-b937-6f0d56ca2700";
+    form:conditions
+      [ a form:SingleCodelistValue ;
+        form:grouping form:Bag ;
+        sh:path rdf:type ;
+        form:conceptScheme <http://lblod.data.gift/concept-schemes/71e6455e-1204-46a6-abf4-87319f58eaa5> ;
+        form:customValue <https://data.vlaanderen.be/id/concept/BesluitType/408a3838-6fcd-4add-937f-dffd9e531b26>
+      ] ;
+    form:hasFieldGroup fieldGroups:f660f439-6f92-4c60-ab02-d6388d52ad79 .
+
+fieldGroups:f660f439-6f92-4c60-ab02-d6388d52ad79 a form:FieldGroup ;
+    mu:uuid "f660f439-6f92-4c60-ab02-d6388d52ad79" ;
+
+    form:hasField
+      # TODO: This probably needs to be a custom field since it requires different options?
+      ### Welk-beslissingsorgaan-nam-het-besluit? ###
+      fields:4c7820f0-4011-4ab4-a16a-e128800e11bc,
```

Comment:
This field doesn't work correctly yet. No "beslissingsorganen" can be found. Is the `path` different for "Erediensten"?